.github/workflows/release.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           environment-name: upload-env
           create-args: >-
             anaconda-client
-      - run: anaconda --token ${{ secrets.ANACONDATOKEN }} upload --user scipp --label main $(ls conda-package-*/*/*.tar.bz2)
+      - run: anaconda --token ${{ secrets.ANACONDATOKEN }} upload --user scipp --label main $(ls conda-package-noarch/*.tar.bz2)
```

Comment:
Why did the path to the package change?

Reply:
Because of 
<img width="477" alt="image" src="https://github.com/scipp/copier_template/assets/12912489/5b32fa9e-091a-4b79-93e3-4ab2f3b2938b">
 in #37